WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -59,9 +59,13 @@ def submit_report_nag(competition)
     @competition = competition
     mail(
       to: competition.delegates.pluck(:email),
-      cc: ["board@worldcubeassociation.org"] + competition.delegates.map { |delegate| delegate.senior_delegate&.email }.uniq.compact.flatten,
+      cc: ["board@worldcubeassociation.org"] + delegates_to_senior_delegates_email(competition.delegates),
       reply_to: "board@worldcubeassociation.org",
       subject: "#{competition.name} Delegate Report",
     )
   end
-end
+
+  private def delegates_to_senior_delegates_email(delegates)
+    delegates.map { |delegate| delegate.senior_delegate&.email }.uniq.compact
+  end
+end
```

Comment:
Missing a newline here. That's why travis is failing (see https://travis-ci.org/thewca/worldcubeassociation.org/builds/292413212#L860)

Reply:
Oops my bad, I wasn't sure. Now I know ^^